stf/model/prediction/prediction.py

```diff
@@ -116,6 +116,12 @@ def make_basecase_forecast(pj, historic_load, overwrite_delay_hours=48):
         basecase_forecast.dropna(inplace=True)
         # rename
         basecase_forecast = basecase_forecast.rename(columns=dict(load="forecast"))
+
+        # Also make basecase for component "other". We do this to make sure the sum of
+        # the basecase of components equals the basecase of the net load
+        # (basecase of wind and pv components is zero)
+        basecase_forecast['forecast_other'] = basecase_forecast['forecast']
```

Comment:
Good to add a comment here! Perhaps you can rephrase this a little bit to make it more clear perhaps. What do you think about something along the line of:

```python
# Also make a basecase forecast for the forecast_other component. This will make a
# simple basecase components forecast available and ensures that the sum of 
# the components (other, wind and solar) is equal to the normal basecase forecast
```